synapse/lib/cell.py

```diff
@@ -58,6 +58,57 @@ class CellApi(s_base.Base):
         assert user
         self.user = user
 
+    def allowed(self, *path):
```

Comment:
Good call.  While we're at it, lets make this async.  It's a backward compat chage since *all* telepath API calls are async to the caller and helps with perf.  When telepath dispatches a method that isn't async, he fires it in an executor which is slightly slower.

Reply:
done